deployment/deployContracts.js

```diff
@@ -31,8 +31,11 @@ async function main() {
     const forkID = deployParameters.forkID || 0;
     const version = '0.0.1';
 
-    const pendingStateTimeout = deployParameters.pendingStateTimeout || (60 * 60 * 24 * 7 - 1);
-    const trustedAggregatorTimeout = deployParameters.trustedAggregatorTimeout || (60 * 60 * 24 * 7 - 1);
+    const pendingStateTimeout = deployParameters.pendingStateTimeout || (60 * 60 * 24 * 7 - 1); // 1 week minus 1
```

Comment:
For local deployment, `npm run deploy:zkEVM:hardhat` a file needed for upgradeability is created `.openzeppelin/unknown-31337.json`.
Second deploy on hardhat does not detect since it looks for `.openzeppelin/harhat.json`.

It does for `goerli`

Reply:
this is and edge case of hardhat network. Library of OZ do not break with this, bc i think they check if the contract is deployed or not, and redeploy it if necessary. Since is an edge case already managed by the OZ lib, i don't think it's worth to add further checks